pom.xml

```diff
@@ -469,6 +475,91 @@
+              <sortArtifactByName>true</sortArtifactByName>
+              <includeTransitiveDependencies>true</includeTransitiveDependencies>
+              <thirdPartyFilename>THIRD-PARTY.txt</thirdPartyFilename>
+              <!-- specify "missing file" licenses for not detected dependencies -->
+              <useMissingFile>true</useMissingFile>
+              <missingFile>src/license/THIRD-PARTY.properties</missingFile>
+              <deployMissingFile>true</deployMissingFile>
+              <failOnMissing>false</failOnMissing>
+              <overrideFile>src/license/THIRD-PARTY-override.properties</overrideFile>
+              <licenseMerges>
+                <!-- Apache License 1.1 https://spdx.org/licenses/Apache-1.1.html -->
+                <licenseMerge>Apache-1.1|Apache Software License, version 1.1</licenseMerge>
+                <!-- Apache License 2.0 https://spdx.org/licenses/Apache-2.0.html -->
+                <licenseMerge>Apache-2.0|Apache 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache Software License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|The Apache Software License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|The Apache License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache 2.0 License</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|ASL, version 2</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License V2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License, version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License, 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache 2</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache Software License, version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License Version 2.0</licenseMerge>
+                <!-- Eclipse Public License 1.0 https://spdx.org/licenses/EPL-1.0.html -->
+                <licenseMerge>EPL-1.0|Eclipse Public License - v 1.0</licenseMerge>
+                <licenseMerge>EPL-1.0|Eclipse Public License 1.0</licenseMerge>
+                <!-- Eclipse Public License 2.0 https://spdx.org/licenses/EPL-2.0.html -->
+                <licenseMerge>EPL-2.0|Eclipse Public License - v 2.0</licenseMerge>
+                <licenseMerge>EPL-2.0|EPL 2.0</licenseMerge>
+                <!-- BSD 2-Clause "Simplified" License https://spdx.org/licenses/BSD-2-Clause.html -->
+                <licenseMerge>BSD-2-Clause|BSD 2-Clause</licenseMerge>
+                <licenseMerge>BSD-2-Clause|BSD 2-Clause License</licenseMerge>
+                <licenseMerge>BSD-2-Clause|The 2-Clause BSD License</licenseMerge>
+                <licenseMerge>BSD-2-Clause|New BSD License</licenseMerge>
```

Comment:
We cannot say fore sure that "New BSD License" will always be BSD-2-Clause.

Reply:
I've removed all licenseMerges.

Reply:
Hm, that's a good question, I had doubts too about this. But when I checked dependency which uses this "New BSD License" https://mvnrepository.com/artifact/org.hamcrest/hamcrest-all/1.3 then following by license's link opens this page https://opensource.org/licenses/bsd-license.php and there is stated as kind of the "The 2-Clause BSD License". Not sure it prove something because all these mess of kind a "open" licenses confuses me.

Reply:
Yes, for hamcrest this is correct.
Some other library my get added to the direct or indirect dependencies in future and if it also uses "New BSD License" it may be incorrect.